iree/tools/BUILD

```diff
@@ -139,6 +139,7 @@ cc_library(
         "//iree/compiler/Translation/SPIRV/XLAToSPIRV",
         "//iree/compiler/Translation/SPIRV/LinalgToSPIRV",
         "@llvm-project//llvm:support",
+        "@llvm-project//mlir:IR",
```

Comment:
This is already pulled in by `":PassesAndDialects", thus no need to add it.

Reply:
Discussed in Discord. Still need this because of inconsistency between Google and GitHub build environment differences. :(